src/vscode-dts/vscode.proposed.attributableCoverage.d.ts

```diff
@@ -10,7 +10,7 @@ declare module 'vscode' {
 		 * file. If set, then {@link TestRunProfile.loadDetailedCoverageForTest}
 		 * should also be defined in order to retrieve detailed coverage information.
 		 */
-		fromTests: TestItem[];
+		fromTests?: TestItem[]; // TODO: `from` feels weird to me. Maybe `sourceTests` or `contributingTests` or even just `tests`
```

Comment:
I think "includesTests" would be a good way to indicate this. That is quite descriptive as to what's going on here